anta/input_models/services.py

```diff
@@ -29,3 +32,24 @@ def __str__(self) -> str:
         Server 10.0.0.1 (VRF: default, Priority: 1)
         """
         return f"Server {self.server_address} (VRF: {self.vrf}, Priority: {self.priority})"
+
+
+class ErrdisableRecovery(BaseModel):
+    """Model for the error disable recovery functionality."""
+
+    model_config = ConfigDict(extra="forbid")
+    reason: ErrdisableReason
+    """Name of the error disable reason."""
+    status: Literal["Enabled", "Disabled"] = "Enabled"
+    """Operational status of the reason. Defaults to 'Enabled'."""
+    timer_interval: int = Field(ge=30, le=86400)
+    """Timer interval of the reason in seconds. Required field in the `VerifyErrdisableRecovery` test."""
```

Comment:
```suggestion
    """Timer interval of the reason in seconds."""
```
This field is not optional so no need to say it's required.

Reply:
Updated